beater/common_handler.go

```diff
@@ -164,12 +164,23 @@ func logHandler(h http.Handler) http.Handler {
 			"remote_address", utility.RemoteAddr(r),
 			"user-agent", r.Header.Get("User-Agent"))
 
+		defer func() {
+			if r := recover(); r != nil {
+				var ok bool
+				if err, ok = r.(error); !ok {
+					err = fmt.Errorf("recovering from %+v", r)
```

Comment:
the wording ends up a little strange with this
```
2019-04-23T14:52:54.229-0400	ERROR	[request]	beater/common_handler.go:173	panic handling request	{"request_id": "111f7740-7338-4a7a-851e-e34a076ba86b", "method": "GET", "URL": "/", "content_length": 0, "remote_address": "127.0.0.1", "user-agent": "curl/7.54.0", "error": "recovering from foo"}
```

maybe `internal server error %+v` ?